src/inferring-outlive-requirements.md

```diff
@@ -0,0 +1,113 @@
+# Infer Outlives Requirements
+
+This feature allows some outlives requirements to be
+inferred by the compiler rather than having to state
+them explicitly.
+
+## Motivation
+
+In the following piece of code, we have the outlives
+relation `T: 'a`; which can be read as T outlives 'a.
+However if we look at the definition of the struct we
+see that the field `bar` is a reference to T with a lifetime
+of 'a. Thus for this code to be valid, the compiler already
+enforces the predicate `T: 'a`.
+
+With this feature the compiler will also infer the outlives
+requirements so that the where-statement becomes optional.
+
+```rust,ignore (pseudo-Rust)
+struct Foo<'a, T>
+  where T: 'a // <-- now optional
+  {
+      bar: &'a T,
+  }
+```
+
+
+For more more information on the motivation and design, please
+see RFC [#44493].
+
+[#44493]: https://github.com/rust-lang/rust/issues/44493
+
+
+## Examples
+
+Code speakers louder than words so here are a few examples
```

Comment:
This sentence is redundant and doesn't read formally.

Reply:
Will update language.